plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/ERDUIConstants.java

```diff
@@ -83,4 +84,7 @@ public class ERDUIConstants {
     public static final int DEFAULT_NOTE_BORDER_WIDTH = 1;
     public static final int DEFAULT_ENTITY_BORDER_WIDTH = 2;
 
+    public static final String ROUTING_MIKAMI = "Mikami-Tabuchi routing algorithm (EXPERIMENTAL)";
+    public static final String ROUTING_SHORTEST_PATH = "Shortest Path";
```

Comment:
Please use an enumeration and use its `name` as key.

Once the Mikami router is no longer experimental and if it was previously chosen, it will revert back to default settings because you will remove the `EXPERIMENTAL` postfix.